### PR TITLE
FAI-11601 | Added full run mode (GitHub)

### PR DESCRIPTION
**Pull request by dbruno21**
## Description

Added full run mode (GitHub)

## Type of change
- [ ] Bug fix
- [X] New feature
- [ ] Breaking change

### CODE REVIEW

File: sources/github-source/src/streams/common.ts [hunk 1]
```diff
@@ -48,6 +64,7 @@ export const RunModeStreams = {
   [RunMode.CopilotEvaluation]: CopilotEvaluationStreamNames,
   [RunMode.Minimum]: MinimumStreamNames,
   [RunMode.Standard]: StandardStreamNames,
+  [RunMode.Full]: FullStreamNames,
```
Comment (chalenge) [inline sources/github-source/src/streams/common.ts:67]: what will be the difference between Full and Standard?

Comment (dbruno21) [reply sources/github-source/src/streams/common.ts:67]: I just saw on the spike docs, that Standard doesn't include ALL.
Full must include all except toggle ones.

Comment (ypc-faros) [reply sources/github-source/src/streams/common.ts:67]: That `Standard` name feels kind of vague to me. Although we're going to have to document the streams included for each mode anyway, so not a big deal I guess.

**Review by ypc-faros — APPROVED** [review]

File: sources/github-source/src/streams/common.ts (hunk 1 above)
Comment (chalenge) [reply sources/github-source/src/streams/common.ts:67]: Fine for now. Need to think more

**Review by chalenge — APPROVED** [review]

**Pull request merged**